tools/quickstart/quickstart/cli.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+
+"""import private key or keystore file as parity account"""
+
+import sys
+import os
+import re
+import json
+from eth_account import Account
+import click
+
+
+class TrustlinesFiles:
+    def __init__(self, password_file, address_file, keystore_file):
+        self.password_file = password_file
+        self.address_file = address_file
+        self.keystore_file = keystore_file
+
+    def store(self, account, password):
+        # Instead of just copying the file, we encrypt it again since the
+        # file itself may use scrypt as key derivative function which
+        # parity can't handle
+        json_account = account.encrypt(password, kdf="pbkdf2")
+
+        os.makedirs(os.path.dirname(os.path.abspath(self.keystore_file)), exist_ok=True)
+        with open(self.keystore_file, "w") as f:
+            f.write(json.dumps(json_account))
+
+        with open(self.password_file, "w") as f:
+            f.write(password)
+
+        with open(self.address_file, "w") as f:
+            f.write(account.address)
+
+
+def import_keystore_file(trustline_files, keystore_input_file):
+    keyfile_dict = json.loads(open(keystore_input_file, "rb").read())
+
+    while 1:
+        password = click.prompt(text="Password", hide_input=True)
+        try:
+            account = Account.from_key(Account.decrypt(keyfile_dict, password))
+            break
+        except ValueError:
+            click.echo("The password you entered is wrong. Please try again.")
```

Comment:
That, or the user did not provide a valid keyfile.

Reply:
I'm not sure if that can really happen after reading the source code, though I also can't rule it out.

 I may provide another PR for it.